lib/cmap/auth/sspi.js

```diff
@@ -24,9 +23,13 @@ class SSPI extends AuthProvider {
       mechanismProperties['gssapiServiceName'] ||
       'mongodb';
 
+    if (Kerberos[kModuleError]) {
+      callback(Kerberos[kModuleError]);
```

Comment:
you need to `return` after calling back. I think this is a copy-paste issue with all of these cases (including Snappy below)